plugins/httpsreq/httpsreqfunc.cpp

```diff
@@ -0,0 +1,126 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2019-2022 Second State INC
+
+#include "httpsreqfunc.h"
+#include <errno.h>
+#include <iostream>
+#include <netdb.h>
+#include <openssl/err.h>
+#include <openssl/ssl.h>
+#include <resolv.h>
+#include <string.h>
+#include <unistd.h>
+
+// Some of the code was taken from this post:
+// https://stackoverflow.com/questions/52727565/client-in-c-use-gethostbyname-or-getaddrinfo
+
+namespace WasmEdge {
+namespace Host {
+
+Expect<void>
+SendData::body([[maybe_unused]] Runtime::Instance::MemoryInstance *MemInst,
```

Comment:
You definitely used `MemInst`. Why do you mark it as `maybe_unused`?

Reply:
I will delete the [[maybe_unused]] here.